client/components/Orders.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchUserOrders } from '../store/orders';
+import { isLoggedIn } from './helperFunctions';
+
+class Orders extends Component {
+  componentDidMount() {
+    const { fetchOrdersThunk, user } = this.props;
+    if (isLoggedIn(user)) fetchOrdersThunk(user.id);
+  }
+
+  render() {
+    const { orders } = this.props;
+    return (
+      <div>
+        <h2>Order History</h2>
+        <table className="table table-striped table-condensed">
+          <thead>
+            <tr>
+              <th>Order Number</th>
+              <th>Date</th>
+              <th>Type</th>
+              <th>Subtotal</th>
+              <th>Shipping</th>
+              <th>Total</th>
+              <th>Status</th>
+              <th />
+            </tr>
+          </thead>
+          <tbody>
+            {orders.map(order => {
+              return (
+                <tr key={order.id}>
+                  <td>{order.id}</td>
+                  <td>{new Date(order.date).toLocaleString()}</td>
+                  <td>{order.type}</td>
+                  <td>${order.subtotal}</td>
+                  <td>${order.shipping}</td>
+                  <td>${order.total}</td>
+                  <td>{order.status}</td>
+
+                  <td>
+                    <Link exact to={`/user/orders/${order.id}`}>
```

Comment:
Should not need `exact` since not in a Route with other links
```suggestion
                    <Link to={`/user/orders/${order.id}`}>
```